<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWindow</class>
 <widget class="QWidget" name="CommandWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>10</y>
     <width>461</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_request">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Запрос</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Команды</string>
   </property>
   <widget class="QComboBox" name="cbCommandFN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
